cli/cmd/org/edit.go

```diff
@@ -23,9 +27,32 @@ func EditCmd(cfg *config.Config) *cobra.Command {
 			}
 			defer client.Close()
 
-			resp, err := client.GetOrganization(ctx, &adminv1.GetOrganizationRequest{Name: args[0]})
+			if !cmd.Flags().Changed("org") {
```

Comment:
We agreed to use default org in cases when `--org` is not passed. Also applied to `rill org edit`.
Also if you decide to prompt in these cases, please pass the default org as default in SelectPrompt.